packages/local_auth/local_auth/windows/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+cmake_minimum_required(VERSION 3.15)
+set(PROJECT_NAME "local_auth")
+set(CPPWINRT_VERSION "2.0.210806.1")
+project(${PROJECT_NAME} LANGUAGES CXX)
+include(FetchContent)
+
+set(PLUGIN_NAME "${PROJECT_NAME}_plugin")
+
+FetchContent_Declare(nuget
+  URL "https://dist.nuget.org/win-x86-commandline/v6.0.0/nuget.exe"
+  URL_HASH SHA256=04eb6c4fe4213907e2773e1be1bbbd730e9a655a3c9c58387ce8d4a714a5b9e1
+  DOWNLOAD_NO_EXTRACT true
+)
+
+find_program(NUGET nuget)
+if (NOT NUGET)
+    message("Nuget.exe not found, trying to download or use cached version.")
+    FetchContent_MakeAvailable(nuget)
+    set(NUGET ${nuget_SOURCE_DIR}/nuget.exe)
+endif()
+
+execute_process(COMMAND
+    ${NUGET} install Microsoft.Windows.CppWinRT -Version ${CPPWINRT_VERSION} -OutputDirectory packages
```

Comment:
Will this install be completely localized to the plugin build, or will it have machine-level effects?

Reply:
The only machine-level side effect of this is the download of CppWinRT's nuget file to Nuget's cache (_~\.nuget\packages_), and the unzipping of it to that same folder, but it should be that, and only that: a cache. Each Flutter project that is built will have its own plugin folder, which will have its own copy of that cached folder locally (that is what the -OutputDirectory parameter is doing there).